app/javascript/App.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link, Route, Routes } from 'react-router-dom';
+import Greeting from './components/Greeting';
+import Home from './components/Home';
+
+function App() {
+  return (
+    <div>
+      <Link to="/greeting">View Greetings</Link>
```

Comment:
- [OPTIONAL] It would have been better if the event to generate the random greetings is added to this element at every click, without the need to go back to home before we can generate a new random greeting.